ui/.betterer.ts

```diff
@@ -1,24 +1,9 @@
 import { typescriptBetterer } from "@betterer/typescript";
 import { regexpBetterer } from "@betterer/regexp";
-import { smaller } from "@betterer/constraints";
-
-const { execSync } = require("child_process");
-
-const jsFileCount = parseInt(execSync(
-  'find . -type f -name "*.js" -not -path "./node_modules/*" | wc -l',
-  { encoding: "utf8" }
-)
-  .replace("\n", "")
-  .trim());
 
 export default {
   "stricter compilation": typescriptBetterer("./tsconfig.json", {
     strict: true,
   }),
   "no TSFixMe types": regexpBetterer("**/*.ts", /(\s*TSFixMe)/i),
-  "migrate js files to ts": {
```

Comment:
Awww.. i thought this was a good reminder if you absentmindedly added a js file instead of a ts. Can we rewrite this somehow?

Reply:
I can try a different approach to counting files perhaps.

Reply:
I think this is actually a bug in the betterer extension, not anything to do with the implementation of the file count. I think it does probably make sense to remove the test until that's resolved so that can get some IDE feedback.